campaigns.py

```diff
@@ -6,52 +6,86 @@
 
 from jinja2 import Template
 
-import vm
+import app
+
+
+env['eagerly_disconnect'] = True
+
+
+APPLICATIONS = ['frontend', 'static']
+CAMPAIGN_CLASSES = ['red', 'black', 'green']
+
 
 def validate_classes(campaign_class):
     """Checks that the campaign class is valid"""
-    if campaign_class in ['red', 'black', 'green']:
+    if campaign_class in CAMPAIGN_CLASSES:
         return campaign_class
     raise Exception, "Invalid class %s, valid values are 'red', 'black', 'green'" % campaign_class
 
 
 @runs_once
-def homepage_template():
-    """Promotes a campaign to the homepage of GOV.UK"""
-    context = {
-        'heading': prompt("Heading for campaign:"),
-        'extra_info': prompt("Extra information for campaign:"),
-        'more_info': prompt("Link for more information:"),
-        'campaign_class': prompt("Campaign class:", validate=validate_classes)
+def set_context():
+    env['context'] = {
+        'heading': prompt("Heading for campaign:", key='heading'),
```

Comment:
I mean that the explicit named argument doesn't need to exist: it could just be:

`prompt("...", 'heading')`